backend/taxHandlers.ts

```diff
@@ -0,0 +1,102 @@
+import { check } from "prettier";
+import { TaxForOrderRes } from "taxjar/dist/util/types";
+import { OrderSubscriptionFragment } from "../generated/graphql";
+import { createOrderTransaction, fetchTaxes } from "./taxjarApi";
+import {
+  CheckoutPayload,
+  ResponseTaxPayload,
+  FetchTaxesPayload,
+  TaxJarConfig,
+  OrderPayload,
+} from "./types";
+
+const _calculateTaxes = async (
+  taxData: FetchTaxesPayload,
+  taxJarConfig: TaxJarConfig
+): Promise<{ data: ResponseTaxPayload }> => {
+  const taxResposne = await fetchTaxes(taxData, taxJarConfig);
+  const taxDetails = taxResposne.tax.breakdown;
+  const shippingDetails = taxDetails?.shipping;
+  let shippingPriceGross = Number(taxData.shipping_amount);
+  let shippingPriceNet = Number(taxData.shipping_amount);
+  let shippingTaxRate = 0;
+
+  if (taxDetails !== undefined) {
+    if (shippingDetails !== undefined) {
+      shippingPriceGross =
+        shippingDetails.taxable_amount + shippingDetails.tax_collectable;
+      shippingPriceNet = shippingDetails.taxable_amount;
+      shippingTaxRate = shippingDetails.combined_tax_rate;
+    }
+  }
+
+  return {
+    data: {
+      shipping_price_gross_amount: String(shippingPriceGross),
+      shipping_price_net_amount: String(shippingPriceNet),
+      shipping_tax_rate: String(shippingTaxRate),
+      // lines order needs to be the same as for recieved payload.
+      lines: taxData.lines.map((line) => {
+        let totalGrossAmount = line.total_amount;
+        let totalNetAmount = line.total_amount;
+        let taxRate = "0";
+        if (taxDetails !== undefined && taxDetails.line_items !== undefined) {
```

Comment:
```suggestion
        let taxRate = "0";

        if (taxDetails !== undefined && taxDetails.line_items !== undefined) {
```
add newlines between constants and if statements for better readability